### DISCUSSION TITLE
Is there a way to prevent the bigwig y axis from rescaling automatically? e.g. specify a fixed scale for the entire assembly?

### DISCUSSION THREAD

**Discussion by ihh** [GMOD/jbrowse-components#4127] [General]
It's a bit disconcerting e.g. for the jbrowse-jupyter tutorial when you move from one window region to another, the scale changes after the pan animation is complete, and it looks like the plotted value has mysteriously changed; only careful attention to the y-axis scale reveals that that is what has actually changed. Is that a configurable option?

**Comment by cmdcolin** [comment]
some of the options for this include setting specific min and max scores, or choosing global autoscale


![image](https://github.com/GMOD/jbrowse-components/assets/6511937/ca01e1f8-2946-4d84-b1be-3e2c054b1261)



these options are also accessible via json config. looks like the following, e.g. setting min and max to -1000 and 1000

```
{
  "type": "QuantitativeTrack",
  "trackId": "mytrack",
  "name": "mytrack",
  "assemblyNames": ["volvox"],
  "adapter": {
    "type": "BigWigAdapter",
    "bigWigLocation": {"uri": "volvox_microarray.bw"}
  },
  "displays": [
    {
      "type": "LinearWiggleDisplay",
      "displayId": "mydisplayid",
      "minScore": -1000,
      "maxScore": 1000
    }
  ]
}
```

**Reply by cmdcolin** [reply]
I'd have to double check how  json configs are specified in e.g. jbrowse-jupyter. will probably be part of doc revamps :)
